### PR TITLE
Chinese traditional (zh_Hant) translation draft

### PR DESCRIPTION
**Pull request by taiansu**
- Create about-zh-TW.md
- Create django.po

### CODE REVIEW

**Comment by fatiherikli** [comment]
We have some questions.

We will setup zh.arguman.org subdomain. Which one should be default, zn-tw or zn-cn?

@tvvocold

**Comment by taiansu** [comment]
@fatiherikli  Thank you so much to bring this up. :heart: Somehow this is a hard question (at least for me), because it's really controversial in the political aspect. And I doubt we can solve this by create a argument map. :laughing: 

Back to the topic, I agree the zh-CN should be the default if you must have a zh.arguman.org, IMHO. But I'm also wondering what's your plan on accessing zh-TW. Could we have the `[zh-]tw.arguman.org` for that?

**Comment by fatiherikli** [comment]
I guess we will setup two subdomain.

ch.arguman.org
tw.arguman.org

Should the content filtered by language? For example, tw.arguman.org will show only zh-tw arguments, is it okay?

**Comment by tvvocold** [comment]
@fatiherikli 
I think it should be:
cn.arguman.org 
tw.arguman.org

**Comment by fatiherikli** [comment]
Sorry, `ch.arguman.org` was a typo :')

**Comment by taiansu** [comment]
cn & tw subdomains sounds great. :heart:  Thank you so much.
For the content filtering things, though I like to have all available contents personally, I'll leave the decision to you. :smile:

File: web/locale/zh_TW/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -0,0 +1,881 @@
...
+msgid "Fallacy Of Previous This"
+msgstr "史學家謬誤"
+
+#: premises/models.py:47
+msgid "Joint Effect"
+msgstr "複合結果"
+
+#: premises/models.py:48
+msgid "Wrong Direction"
+msgstr "倒因為果"
```
Comment (taiansu) [inline web/locale/zh_TW/LC_MESSAGES/django.po]: Should be "倒果為因"

**Comment by Arthur2e5** [comment]
Since the zh_CN one is now called by its script name zh_Hans/ 简体 (simplified), for TW you may want to call it by the script name too: zh_Hant / 繁体 (traditional).

**Comment by Arthur2e5** [comment]
By the way I consider using a big zh domain is a good idea, since the two script variants (and, hmm, CN, SG, HK, TW, 4 locales) are largely interchangeable using simple conversion methods like ByVoid/OpenCC or the one that Wikipedia uses.

**Comment by Arthur2e5** [comment]
And for language selection, why not consult the Accept-Language header from the browser?

Basically zh_Hant, zh_TW and zh_HK can be put into a bucket, and zh_Hans, zh_CN and zh_SG into the other.

**Comment by Arthur2e5** [comment]
>By the way I consider using a big zh domain is a good idea, since the two script variants (and, hmm, CN, SG, HK, TW, 4 locales) are largely interchangeable using simple conversion methods like ByVoid/OpenCC or the one that Wikipedia uses.

For a real-world example, http://ch.arguman.org/xi-ma-hui-you-zhu-tai-wan-she-hui-fa-zhan is written in zh_Hant (script).

File: web/locale/zh_TW/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -0,0 +1,881 @@
...
+"Project-Id-Version: Arguman\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2015-10-13 08:22+0000\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: Jorge Barata <jorge.barata.gonzalez@gmail.com>\n"
+"Language: es\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
```
Comment (Arthur2e5) [inline web/locale/zh_TW/LC_MESSAGES/django.po]: The plural form for Chinese is `nplurals=1; plural=0;` unless you want to handle some pronouns like 他們.

File: web/locale/zh_TW/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -0,0 +1,881 @@
...
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: Arguman\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2015-10-13 08:22+0000\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: Jorge Barata <jorge.barata.gonzalez@gmail.com>\n"
+"Language: es\n"
```
Comment (Arthur2e5) [inline web/locale/zh_TW/LC_MESSAGES/django.po]: zh_TW

**Comment by taiansu** [comment]
@Arthur2e5 Hi, thanks for your review. Fix as your suggestion. Should I also change the file name to zh_Hant correspondly?

**Comment by Arthur2e5** [comment]
Not sure yet. Maybe…

**Pull request closed without merging**